src/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -20,6 +22,10 @@
     <ProjectJson>unix/project.json</ProjectJson>
     <ProjectLockJson>unix/project.lock.json</ProjectLockJson>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(TargetGroup)' == 'net46' and '$(ProjectJson)' == '' ">
+    <ProjectJson>net46/project.json</ProjectJson>
```

Comment:
ok. thx for the explanation. I can make that small change when I do another commit.

The core problem of the actual build/packaging not working requires someone like @ericstj to help troubleshoot.